### ISSUE TITLE
Colorado route E-470

### ISSUE THREAD

**Issue by Pengor** [ZeLonewolf/openstreetmap-americana#248]
Colorado's toll route E-470 has a separate shield than the state DOT's numbered highways. As such, its `network` tag needs to be updated and a separate shield should be added to this project for that network.

See also discussion on OSM US Slack Workspace [here](https://osmus.slack.com/archives/C01V02K52UX/p1647884121104639).

@amcginni has drawn a shield for this.

**Comment by amcginni** [comment]
https://www.openstreetmap.org/changeset/118826107 changes the `network` tag to `US:CO:E470`.  This name was chosen because the name of the toll road operator is "E-470 Public Highway Authority."  The dash in the `ref` tag was also removed to match the signs.

I am going to hold off on submitting a PR for the E470 shields until #249 is finalized.  I think it would look good if the logo in those shields was the same height as the Colorado flag in the state route shields.  For example, the following mock-up uses the same vertical proportions for the E470 shield as the Colorado shield.  I want to replicate this effect.
![image](https://user-images.githubusercontent.com/46615954/159736207-6f9799a3-ba95-44df-9490-78d86b776ef7.png)

**Comment by ZeLonewolf** [comment]
Another option to consider is simply a square shield with just the mountain icon.  That would take some artistic liberty but might be nicer looking than the really wide version.

**Comment by 1ec5** [comment]
Would the mountain be recognizable enough on its own? Some people may depend on the number enough that they may mistake SH&nbsp;470 for E-470.

**Comment by ZeLonewolf** [comment]
Yeah I have no idea which portion of the shield is more important.  The shield gives basically equal weight to either part, and the graphic above drops the mountain down to unrecognizable.

![image](https://user-images.githubusercontent.com/3254090/159837187-50f271ee-185e-4bd3-b49c-f5e910c10c0c.png)
